setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+requirements = [
+      'gym',
+      'chainer',
+      'numpy',
+      'Pillow']
```

Comment:
Is there any way to prevent possible future errors based on forgetting to put crucial requirements here in addition to putting them into `requirements.txt`? /cc @Kolpa @milanpro